Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/view/CovidCertificateValidationResultView.kt

```diff
@@ -82,4 +82,8 @@ class CovidCertificateValidationResultView @JvmOverloads constructor(
     override fun setOnClickListener(l: OnClickListener?) {
```

Comment:
Do we need this one then? I don't think it's used anywhere else and it's basically replaced by `setNavigationOnClickListener`